common/views/components/IIIFViewer/ViewerSidebar.tsx

```diff
@@ -114,6 +116,7 @@ const AccordionItem = ({
         </button>
       </AccordionInner>
       <AccordionInner
+        id={title}
```

Comment:
Could be worth [adding this](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/utils/string.ts#L1) incase there's a two word title.